templates/Web/Projects/ReactDefault/frontend/scripts/publish.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
```

Comment:
@sibille @dgomezc @prupipho  Just seen this when trying to execute task EJECT in react. Seems that it can create the files in the script folder because it already exists...

![image](https://user-images.githubusercontent.com/1333036/97951619-12f34000-1d9b-11eb-8230-2f72eb8ef090.png)

Reply:
I think we're fine here, eject is for advanced developers and the error contains instructions on how to proceed.